pisp_app/lib/Screens/new_payment.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
+import 'package:pispapp/MockData/account.dart';
+import 'package:pispapp/Screens/lookup_payee.dart';
+import 'package:pispapp/Screens/phone_number.dart';
+
+class NewPayment extends StatefulWidget {
+  @override
+  _NewPaymentState createState() => _NewPaymentState();
+}
+
+class _NewPaymentState extends State<NewPayment> {
+  _NewPaymentState() {
+    final List<Account> accounts = getMyDummyAccounts();
+    _accounts = accounts.where((Account element) => element.linked).toList();
+  }
+  
+  List<Account> _accounts = <Account>[];
+  Account selectedAccount;
+
+  String phoneNumber;
+  String phoneIsoCode;
+
+  void onPhoneNumberChange(
+      String number, String internationalizedPhoneNumber, String isoCode) {
+    setState(() {
+      phoneNumber = number;
+      phoneIsoCode = isoCode;
+    });
+  }
+
+  
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        child: Column(
+      children: <Widget>[
+        const SizedBox(height: 25),
+        const Text(
+          'Choose an account',
+          style: TextStyle(
+            fontSize: 18.0,
+          ),
+        ),
+        const SizedBox(height: 16),
+        DropdownButton<Account>(
+          hint: const Text('Select Account'),
+          value: selectedAccount,
+          items: _accounts.map((Account account) {
+            return DropdownMenuItem<Account>(
+              value: account,
+              child: Row(
+                children: <Widget>[
+                  Text(
+                    account.alias,
+                    style: TextStyle(color: Colors.black),
+                  ),
+                ],
+              ),
+            );
+          }).toList(),
+          onChanged: (Account account) {
+            setState(() {
+              selectedAccount = account;
+            });
+          },
+        ),
+        const SizedBox(height: 60),
+        const Text(
+          'Payee Phone Number',
+          style: TextStyle(
+            fontSize: 16.0,
+          ),
+        ),
+        const SizedBox(height: 16),
+        Padding(
+          padding: const EdgeInsets.fromLTRB(15.0, 0.0, 15.0, 0.0),
+          child: PhoneNumberInput(onPhoneNumberChange),
+        ),
+        Expanded(
+          child: Padding(
+            padding: const EdgeInsets.all(22.0),
+            child: Align(
+                alignment: Alignment.bottomRight,
+                child: ClipOval(
+                  child: Material(
+                    color: Colors.green, // button color
+                    child: InkWell(
+                      splashColor: Colors.white, // inkwell color
+                      child: SizedBox(
+                          width: 56,
+                          height: 56,
+                          child: Icon(Icons.arrow_forward)),
+                      onTap: () async {
```

Comment:
Also, instead of waiting for the user to tap a button like this, I think it would be better to use a `TextEditingController` so that we can display the suggestions as the user types in a phone number (or contact name).

Reply:
We could actually just maintain a contact list and let the user select from that or type a new number (I think that is what gpay does). If we do it that way, we could remove the dependency to `international_phone_input` and also remove the `LookupPayee` screen.

Or perhaps, if you want to keep it, we can just add option for user to search by name instead of typing the number directly. Because I think it is pretty difficult to remember a phone number.

Reply:
Accessing contact list is forbidden as of now as it is user data. JJ asked us to stick to phone as of now.

Reply:
Oh okay got it, but you still maintain the hardcoded contact list tho... so I guess we can still maintain an in-app contact list (?, not synced with the phone contacts), but yeah I think it's just a good-to-have feature and we can save it for another time.